client/src/components/gallery/Strip.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  margin-top: 4px;
+  position: relative;
+  height: 69px;
+  margin-bottom: 4px;
+  z-index: 40;
+`;
+const StripScroll = styled.div`
+  left: ${props => props.stripPos};
+  width: 1562px;
+  position: absolute;
+`;
+
+const HighlightSelector = styled.div`
+  position: absolute;
+  left: ${props => props.selectorPos}
+  background-image: url(https://store.steampowered.com/public/images/v5/game_highlight_activethumb.png);
```

Comment:
Not high priority but graphics shouldn't be directly linked to Steam. Either put on own server or on S3 later.